src/test/java/com/puppycrawl/tools/checkstyle/DetailNodeTreeStringPrinterTest.java

```diff
@@ -37,10 +38,11 @@
 import com.puppycrawl.tools.checkstyle.api.LocalizedMessage;
 import com.puppycrawl.tools.checkstyle.api.TokenTypes;
 
-public class DetailNodeTreeStringPrinterTest {
+public class DetailNodeTreeStringPrinterTest extends BaseCheckTestSupport {
```

Comment:
Since we are changing this, can we use the new `AbstractModuleTestSupport` class instead? We are switching over everything to these new classes.
See commits under #4592 .